src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -105,7 +109,7 @@ public static AuditConfig from(final Settings settings) {
      * Audit logger will use these settings to determine what audit logs are to be generated.
      */
     public static class Filter {
-        private static final Filter DEFAULT = Filter.from(Settings.EMPTY);
+        public static final Filter DEFAULT = Filter.from(Settings.EMPTY);
```

Comment:
it can be kept private

Reply:
Made it visible for testing as one of the tests is using it